packages/core/plugins/scoping.js

```diff
@@ -22,44 +22,48 @@ module.exports = (css, result) => {
         keyframes = Object.create(null),
         globals   = Object.create(null),
         parser, current, lookup;
-    
+
     parser = processor(function(selectors) {
+        const pseudos = [];
+
         selectors.walkPseudos(function(node) {
```

Comment:
Add a comment referencing either #277 or https://github.com/postcss/postcss-selector-parser/pull/105 please so I can recall why the code is doing something that seems weird here.